File: WpfApp/Data/Cargo.cs
#nullable enable
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Text.RegularExpressions;

namespace SqlOrganize.Sql.FinesApp.Data
{
    public partial class Cargo : Entity
    {

        public Cargo()
        {
            _entityName = "cargo";
            _db = Context.db;
            Default();
        }

        #region id
        protected string? _id = null;
        public string? id
        {
            get { return _id; }
            set { if( _id != value) { _id = value; NotifyPropertyChanged(nameof(id)); } }
        }
        #endregion

        #region descripcion
        protected string? _descripcion = null;
        public string? descripcion
        {
            get { return _descripcion; }
            set { if( _descripcion != value) { _descripcion = value; NotifyPropertyChanged(nameof(descripcion)); } }
        }
        #endregion

        #region Designacion_ (ref designacion.cargo _m:o cargo.id)
        public ObservableCollection<Designacion> Designacion_ { get; set; } = new ();
        #endregion

    }
}

File: Model/Cargo.php
<?php

namespace Fines2;

use SqlOrganize\Sql\Entity;
use Exception;
use DateTime;

class Cargo extends Entity
{

    public function __construct()
    {
        $this->_entityName = "cargo";
        $this->_db = \SqlOrganize\Sql\DbMy::getInstance();
        $this->setDefault();
    }

    /** @var string|null */
    public ?string $descripcion = null;

    /** @var string|null */
    public ?string $id = null;

    /** @var int|null */
    public ?int $Designacion_Count = null;

    /** @var Designacion[] (ref designacion.cargo _m:o cargo.id) */
    public array $Designacion_ = [];

}
